### PR TITLE
avoid making display calls while building docs

### PR DESCRIPTION
**Pull request by mjaskula**
The doc generation build is broken by the most recent merge. That code added the ability to rotate the display, which happens on Display creation, which happens when you import europi.py. Since the doc generation system mocks out the ssd1306 library we can't make any calls to it while importing the europi lib.

### CODE REVIEW

**Comment by roryjamesallen** [comment]
I don't fully understand how the docs work, but the change matches what you say needs to happen so LGTM if it works!

**Review by chrisib — APPROVED** [review]
Tested on my module; doesn't break anything on my end.

**Pull request merged**
